modules/gapi/src/executor/gexecutor.cpp

```diff
@@ -398,6 +481,14 @@ void cv::gimpl::GExecutor::run(cv::gimpl::GRuntimeArgs &&args)
     }
 }
 
+void cv::gimpl::GExecutor::runImpl() {
+    for (auto &op : m_ops)
+    {
+        // (5) and  (6)
```

Comment:
I think this comment should stay in `GExecutor::run()` near `runImpl()` (currently, on string 475)

Reply:
fixed